packages/infra/lib/app-config-stack.ts

```diff
@@ -36,7 +38,7 @@ export function createAppConfigStack(
       configurationProfileId: appConfigOSSProfile.ref,
       contentType: "application/json",
       content: JSON.stringify({
-        version: "1",
+        version: "2",
```

Comment:
This might override an existing version. Not sure what the best practice is here, but maybe create a new class of version, like every time we add/remove a FF we create a prefix so we can reset the counter to 1? Thoughts?

Reply:
I played around in the GUI and creating new versions doesnt override old ones or existing ones. I think this is safe as a practice for now. The GUI also shows each version as 1, 2, 3, etc. even if the label given is different. For example:
<img width="1566" alt="Screenshot 2024-01-10 at 16 03 27" src="https://github.com/metriport/metriport/assets/70026960/265062ec-03f7-4c0b-b320-93b93c2004ee">

So I think this approach of incrementing linearly actually makes the most sense for when we examine in gui. I will add a comment so next dev who updates knows though that this is how we do it

Reply:
I tried removing the version from the JSON - it's redundant, we already have a version defined by AWS when we create a new one, and it works:

![image](https://github.com/metriport/metriport/assets/2132564/c17dde76-d589-40f8-9910-9cf4418a289c)

So I suggest we just remove the `version` prop from the JSON.